1-development/user-changes-challenge/__tests__/get-tests.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment node
+ */
+
+const moment = require('moment')
+const assert = require('assert')
+const axios = require('axios')
+const utils = require('../utils/utils')
+
+describe("Testing with valid input", function() {
+    it("Should return status code 200 and body length is 0", function (done) {
+        let id = 347548679
+        let minDate = '2030-10-10'
+        let maxDate = '2030-10-20'
+
+        let url = utils.getURL(id, minDate, maxDate);
+        axios
+            .get(url)
+            .then(response => {
+                assert.strictEqual(response.status, 200)
+                assert.strictEqual(response.data.body.length, 0)
```

Comment:
Why only check for status and length when the endpoint is used to return a collection of changed fields?

Reply:
I've use that method in the module ```post-tests.js``` and in the unit tests that are present in the module ```compare-tests.js```.
In this case I only wanted to check if the server returned an empty array (because there are no values in the database with the ```userID``` and ```start/end dates``` that I used to test).